TROUBLESHOOTING.md

```diff
@@ -148,9 +148,16 @@ mvn quarkus:dev -Djvm.args="-agentpath:/path/to/async-profiler/build/libasyncPro
 
 You can also configure the `jvm.args` system property directly inside the `quarkus-maven-plugin` section of your pom.xml.
 
-## Printing build step execution time
+## Analysing build steps execution time
 
-When trying to debug startup performance it is convenient to log build steps execution time. This can be achieved by adding the following system property: `-Dquarkus.debug.print-startup-times=true`.
+When trying to debug startup performance, it is convenient to log build steps execution time. 
+This can be achieved by adding the following system property: `-Dquarkus.debug.print-startup-times=true` on dev mode or when launching the JAR.
```

Comment:
```suggestion
This can be achieved by adding the following system property: `-Dquarkus.debug.print-startup-times=true` in dev mode or when launching the JAR.
```